templates/kobotoolbox/0/docker-compose.yml

```diff
@@ -108,7 +108,7 @@ services:
     image: kartoza/kobotoolbox_postgres:latest
     environment:
       - ENKETO_API_TOKEN=${ENKETO_API_TOKEN}
-      - DJANGO_SECRET_KEY="${DJANGO_SECRET_KEY}"
+      - DJANGO_SECRET_KEY=${DJANGO_SECRET_KEY}
```

Comment:
Django Secret Key needs to be escaped, because it may contain weird character.
So, let it use: "${DJANGO_SECRET_KEY}" (with quotes)

Reply:
Also do it for others.